llvm/lib/Target/Xtensa/Xtensa.td

```diff
@@ -18,9 +18,15 @@ include "llvm/Target/Target.td"
 // Subtarget Features.
 //===----------------------------------------------------------------------===//
 def FeatureDensity : SubtargetFeature<"density", "HasDensity", "true",
-                    "Enable Density instructions">;
+                                      "Enable Density instructions">;
 def HasDensity : Predicate<"Subtarget->hasDensity()">,
-                     AssemblerPredicate<(all_of FeatureDensity)>;
+                 AssemblerPredicate<(all_of FeatureDensity)>;
+
+def FeatureWindowed : SubtargetFeature<"windowed", "HasWindowed", "true",
+                                       "Enable Xtensa Windowed Register option">;
```

Comment:
Why is this an "option". Is this terminology used in the ISA manual? This is a subtarget featur e

Reply:
The ISA summary refers to these extensions as options, calling this particular one "Windowed Register Option".

Reply:
I created special file XtensaFeatures.td for Xtensa ISA extensions/features.